packages/dht/protos/DhtRpc.proto

```diff
@@ -17,7 +17,10 @@ service DhtNodeRpc {
 service RouterRpc {
   rpc routeMessage (RouteMessageWrapper) returns (RouteMessageAck);
   rpc forwardMessage (RouteMessageWrapper) returns (RouteMessageAck);
-  rpc findRecursively (RouteMessageWrapper) returns (RouteMessageAck);
+}
+
+service FindRpc {
+  rpc find (RouteMessageWrapper) returns (RouteMessageAck);
```

Comment:
this could be called `routeFindRequest` to better explain what is happening.

Reply:
Renamed